File: fork-runner/src/main/java/com/shazam/fork/injector/stat/TestExecutionReporterInjector.java
package com.shazam.fork.injector.stat;

import com.shazam.fork.stat.TestExecutionReporter;

public class TestExecutionReporterInjector {
    private static final TestExecutionReporter instance = new TestExecutionReporter();

    public static TestExecutionReporter testExecutionReporter(){
        return instance;
    }
}

File: fork-runner/src/main/java/com/shazam/fork/injector/stat/TestExecutionReporterInjector.kt
package com.shazam.fork.injector.stat

import com.shazam.fork.injector.GsonInjector.gson
import com.shazam.fork.injector.system.FileManagerInjector.fileManager
import com.shazam.fork.stat.TestExecutionReporter

object TestExecutionReporterInjector {
    @JvmStatic
    fun testExecutionReporter(): TestExecutionReporter {
        return TestExecutionReporter(fileManager(), gson())
    }
}
